tripal/includes/TripalEntityController.inc

```diff
@@ -688,14 +688,19 @@ class TripalEntityController extends EntityAPIController {
           // attach then we will ignore it. It can only be set by providing
           // the id in the $field_id array handled previously.
           else {
-            
-            // Do not load fields that are not auto attached.  Instead set
-            // their value to an empty string and set the #processed key to
-            // FALSE.
+
+            // Does the field instance have an 'auto_attach' setting?
+            $auto_attach = FALSE;
```

Comment:
Just a note... I changed these lines of code to hopefully prevent problems in the future if a developer forgets to set the 'auto_attach'  we need it to have a default.